blackmarket/UI/Common/SearchBar.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  SearchBar.swift
+//  blackmarket
+//
+//  Created by Daniel Parra on 12/6/22.
+//
+
+import SwiftUI
+
+struct SearchBar: View {
+    
+    @Binding var text: String
+    @State private var isEditing = false
+ 
+    var body: some View {
+        ZStack(alignment: .trailing) {
+            TextField(LocalizedString.SearchBarScreen.searchBarPlaceholder, text: $text)
+                .frame(height: UI.SearchBar.TextField.height)
+                .padding(UI.SearchBar.TextField.padding)
+                .padding(.horizontal, UI.SearchBar.TextField.horizontalPadding)
+                .background(.white)
+                .cornerRadius(UI.Defaults.cornerRadius)
+                .padding(.horizontal, UI.SearchBar.horizontalPadding)
```

Comment:
any reason why you are applying the padding operator multiple times for the same edges?
Also, pls be explicit when applying different padding to edges , like:
```
.padding(.vertical, UI.SearchBar.TextField.verticalPadding)
.padding(.horizontal, UI.SearchBar.TextField.horizontalPadding)
```